config/initializers/devise.rb

```diff
@@ -294,6 +294,12 @@
 
   config.jwt do |jwt|
     jwt.secret = ENV['DEVISE_JWT_SECRET_KEY']
+    jwt.dispatch_requests = [
+      ['POST', %r{api/v1/auth/login}]
+    ]
+    jwt.revocation_requests = [
+      ['DELETE', %r{api/v1/auth/logout}]
+    ]
```

Comment:
I don't believe these should be necessary given how devise is configured in the routes.

Reply:
I can take it out. This was me just trying to get the sign up token working. It was something suggested by the creator of the devise jwt gem we're using. It's not relevant to this ticket at any rate.

Reply:
Yeah. If we do end up needing to add it back in, I'd like to walk through it with you. This shouldn't be required, and hasn't been in my experience with hitting this endpoint through Postman.

Reply:
gotcha. I should run stuff through Postman more. For some reason I really dislike Postman and I can't say why . . .